Sources/NIOCrashTester/CrashTests+ByteBuffer.swift

```diff
@@ -0,0 +1,27 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2020 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import NIO
+import Foundation
```

Comment:
Seems unnecessary? (and elsewhere)

Reply:
(There isn't a linter which could fail if an import isn't used, is there? If there is I'd love to know -- Foundation imports creeping into more files than necessary is a bit annoying to me in my work :-/)

Reply:
removed the unused imports

Reply:
I see so you test it based on what's linked hm... Was hoping there's some linter that per source could check it and I could "permit" it sometimes. No worries, just asking